test/insured-vesting-v1/insured-vesting-v1.test.ts

```diff
@@ -432,11 +430,13 @@ describe("InsuredVestingV1", () => {
         await expectRevert(async () => insuredVesting.methods.addFunds(1).send({ from: user1 }), Error.EmergencyReleased);
       });
 
-      it("not enough XCTD balance for deposited USDC", async () => {
-        const amount = 100_000_000;
-        await insuredVesting.methods.setAllocation(user1, await mockUsdc.amount(amount)).send({ from: deployer });
-
-        await expectRevert(async () => insuredVesting.methods.addFunds(await mockUsdc.amount(amount)).send({ from: user1 }), "ERC20: insufficient allowance");
+      it("fails if user does not have enough balance", async () => {
+        const amount = FUNDING_PER_USER + 1;
+        await insuredVesting.methods.setAllocation(user1, await usdc.amount(amount)).send({ from: deployer });
+        await expectRevert(
+          async () => insuredVesting.methods.addFunds(await usdc.amount(amount)).send({ from: user1 }),
+          "ERC20: transfer amount exceeds allowance"
```

Comment:
Also I believe the test was incorrectly titled before

Reply:
Any JS libraries where these messages are exported as constants?

Reply:
not sure. even if there were, still I guess USDC on ethereum could have its own?

I'll just do a comparison on etherscan to make sure it indeed differs from the standard so that this change is justified

Reply:
Yes - USDC (proxy impl.) has the updated error [here] (https://etherscan.io/address/0xa2327a938febf5fec13bacfb16ae10ecbc4cbdcf#code)

Whereas our previous implementation of OpenZeppelin's [ERC20] (https://github.com/OpenZeppelin/openzeppelin-contracts/commit/b425a722409fdf4f3e0ec8f42d686f3c0522af19#diff-fa792f7d08644eebc519dac2c29b00a54afc4c6a76b9ef3bba56c8401fe674f6L314) had a different error - so this is fine. (Btw note I've linked to a commit since they are now using custom errors to revert, as we do)